src/adagrams.js

```diff
@@ -1,15 +1,219 @@
+// HELPER FUNCTIONS
+
+/*
+This function uses the built in method 'reduce' to reduce the array based on the parameters supplied.
+It looks to see if the letter is in the array and, if it is, it adds 1 to the return value.
+*/
+export const countTimes = (arr, val) => arr.reduce((a, v) => (v === val ? a + 1 : a), 0);
+
+// This function loads the winner into an object
+export const loadWinner = (word, score) => {
+  let winner = {};
+
+  winner.word = word;
+  winner.score = score;
+  
+  return winner;
+};
+
+// This function handles any tie scores
+export const tieBreaker = maxScore => {
+  let smallestWordLen = 1000;
+  let winner = {};
+  let nxtWord, winWord = '';
+  let nxtScore, winScore = 0;
+  
+  for (let i = 0; i < maxScore.length; i++) {
+      nxtWord = maxScore[i][0];
+      nxtScore = maxScore[i][1];
+
+      if (nxtWord.length === 10) {
+          winWord = nxtWord;
+          winScore = nxtScore;
+          break;
+      } else if (nxtWord.length < smallestWordLen) {
+          smallestWordLen = nxtWord.length;
+          winWord = nxtWord;
+          winScore = nxtScore;
+      } else {
+          continue;    
+      };
+  };
+
+  winner = loadWinner(winWord, winScore);
+  return winner;
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letterPool = {
```

Comment:
you could also move your letterPool and letterVals into a separate file and import the constants to use in this file for more organization.